docs/getting-started/tutorials/auth-architecture/static-api-key.md

```diff
@@ -0,0 +1,61 @@
+---
+id: static-api-key
+title: Use a static API Key to secure your endpoints
+sidebar_label: Static API Key
+---
+
+The simplest authentication method you can implement with Mia-Platform is using the [API Keys](/development_suite/api-console/api-design/api_key.md) offered by our API Gateway (both [Nginx](/runtime_suite/api-gateway/10_overview.md) or [Envoy](/runtime_suite/envoy-api-gateway/overview.md)) to secure your APIs.
+
+## Definition
+ 
+Basically, you can define a static client key for each client that needs to consume the APIs of your project. You can then decide which client has access to which APIs by configuring the correct group expression in the [Endpoints](/development_suite/api-console/api-design/endpoints.md#manage-the-security-of-your-endpoints) section.
+
+![Scenario1](./img/auth-scenario1.png)
+
+The involved microservice of the flow at runtime are:
+1. `API Gateway`: Mia-Platform plugin available in [Nginx](/runtime_suite/api-gateway/10_overview.md) or [Envoy](/runtime_suite/envoy-api-gateway/overview.md)
+2. [`Authorization Service`](/runtime_suite/authorization-service/10_overview.md): Mia-Platform plugin
+3. The microservice connected to the endpoint 
+
+The picture above illustrates the auth flow at runtime:
+1. The client calls the endpoint of your project, placing their own API key in the `client-key` header (or in the `mia_client_key` cookie).
+2. The API gateway resolves the client type from the `client-key` header. Then, it calls the Authorization Service which is in charge to verify if the resolved client type has access to the requested endpoint.
+3. If the client type verification performed by the authentication service is successful, the API Gateway forwards the API call the right microservice of the project.
+
+:::warning
+The authentication method suggested in this scenario is straightforward but comes with a lower level of security. Therefore, we recommend using it only in use cases where APIs are not exposed on the public internet or in situations where security is not a critical aspect of the project (e.g., MVPs, internal projects, etc.).
```

Comment:
Maybe this message should be `:::danger` and be on top. An API Key should not be used for securing endpoints 😄

Reply:
I agree. Done as suggested